src/javaharness/java/arcs/api/IdGenerator.java

```diff
@@ -0,0 +1,32 @@
+package arcs.api;
+
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+ /**
```

Comment:
I don't think these comments are formatted right, it should be

/**
   *
   *
   */

Reply:
copy&paste messed them up. fixed now.